lib/gamera/modules/visitable.rb

```diff
@@ -8,7 +8,7 @@ module Visitable
     #
     # @param fail_on_redirect [Boolean] Whether to fail if the site redirects to a page that does not match the url_matcher regex
     # @raise [WrongPageVisited] if the site redirects to URL that doesn't match the url_matcher regex and fail_on_redirect is true
-    def visit(fail_on_redirect = true)
+    def visit(fail_on_redirect: true)
```

Comment:
Reminded me to update the README example, thanks. We probably do want a `CHANGES.md` eventually but I'm not sure this is in wide enough use yet to warrant it.